components/CancellationList.vue

```diff
@@ -162,9 +173,24 @@ export default {
           .collection("reports")
           .add(this.report)
           .then((this.showDialog.report = false))
-          .then(() => console.log("Reported", this.report));
+          .then(() => {console.log("Reported", this.report);
+            this.snackBarData = {
```

Comment:
this will need to be updated when moving `snackBarData` to `data` to be reactive.